product/src/main/generated/required-productized-camel-artifacts.txt

```diff
@@ -109,6 +109,7 @@ camel-webhook
 camel-xml-io
 camel-xml-io-dsl
 camel-xml-jaxb
+camel-xml-jaxp
```

Comment:
It seems that  `camel-4.0.0-branch` does not include `camel-xml-jaxp`. So it could fail to build because of missing the product artifact. 

Can you check it first?

Reply:
It is being productized (maybe as its a transitive of something else):

https://github.com/jboss-fuse/camel/blob/camel-4.0.0-branch/parent/pom.xml#L632-L636

https://indy.psi.redhat.com/browse/maven/hosted/pnc-builds/org/apache/camel/camel-xml-jaxp/4.0.0.redhat-00012/

I could add it explicitly to `required-productized-camel-artifacts.txt` though.

Reply:
OK, nice catch! so it looks we are good here. And adding it in `required-productized-camel-artifacts.txt` looks better.

Reply:
https://github.com/jboss-fuse/camel/pull/2286